vars/main.yml

```diff
@@ -1 +1 @@
-# Vars for wipefs
+#Vars for disk_wipe
```

Comment:
If there are no vars, please delete this file.

Reply:
removed

Reply:
removed